qa/common/infer_util.py

```diff
@@ -62,7 +62,15 @@ def _range_repr_dtype(dtype):
     return dtype
 
 
+def _prepend_string_size(input_values):
```

Comment:
_serialize_byte_tensor_list ? Also, we are passing ouput0_values to this function in L152

Reply:
Right. The variable names should be generic